specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/examples/ManagedHsm_Delete.json

```diff
@@ -7,7 +7,11 @@
   },
   "responses": {
     "200": {},
-    "202": {},
+    "202": {
+      "headers": {
+        "Location": "https://some.endpoint.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hsm-group/providers/Microsoft.KeyVault/managedHSMs/hsm1?api-version=2022-07-01&kv-operation=abJjb2RkIjoiAGVsZXRlTWFuYWdlZEhzbUFzeW5jYm9"
```

Comment:
Can you please confirm from code to see if RP returns this header in 2020-04-01-preview version? Just to make sure service code and swagger are in sync.

Reply:
Reverting this change as RP does not return the location header for this version.